Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -442,6 +442,8 @@ std::optional<Inspector::Protocol::CSS::PseudoId> InspectorCSSAgent::protocolVal
         return Inspector::Protocol::CSS::PseudoId::WebKitScrollbarTrackPiece;
     case PseudoId::WebKitScrollbarCorner:
         return Inspector::Protocol::CSS::PseudoId::WebKitScrollbarCorner;
+    case PseudoId::InternalWritingSuggestions:
+        return Inspector::Protocol::CSS::PseudoId::InternalWritingSuggestions;
```

Comment:
i actually wonder if we want to expose this in Web Inspector 🤔

if it's not something that developers can directly interact with (i.e. creating their own `foo::-internal-writing-suggestion { ... }` CSS declaration) then it may not be useful to show anything about it, but otherwise it kinda seems more like an internal implementation detail that developers might get confused about

if we do really want to expose this then you'd also need to add support for this in `WI.CSSManager.PseudoSelectorNames` and `WI.CSSManager.displayNameForPseudoId`

Reply:
ya I don’t want to expose this; in this case, should I just do `return { }`?

Reply:
yeah i think that should work. and also
please remove it from `Source/JavaScriptCore/inspector/protocol/CSS.json` too :)